www/gatsby-node.js

```diff
@@ -85,661 +35,47 @@ exports.createPages = ({ graphql, actions, reporter }) => {
-          })
-          const tutorialIndex = flattenedTutorials.findIndex(findDoc, {
-            link: slug,
-          })
-          const contributingIndex = flattenedContributing.findIndex(findDoc, {
-            link: slug,
-          })
-
-          // add values to page context for next and prev page
-          let nextAndPrev = {}
-          if (docIndex > -1) {
-            nextAndPrev.prev = getSibling(docIndex, flattenedDocs, `prev`)
-            nextAndPrev.next = getSibling(docIndex, flattenedDocs, `next`)
-          }
-          if (tutorialIndex > -1) {
-            nextAndPrev.prev = getSibling(
-              tutorialIndex,
-              flattenedTutorials,
-              `prev`
-            )
-            nextAndPrev.next = getSibling(
-              tutorialIndex,
-              flattenedTutorials,
-              `next`
-            )
-          }
-          if (contributingIndex > -1) {
-            nextAndPrev.prev = getSibling(
-              contributingIndex,
-              flattenedContributing,
-              `prev`
-            )
-            nextAndPrev.next = getSibling(
-              contributingIndex,
-              flattenedContributing,
-              `next`
-            )
-          }
-
-          createPage({
-            path: localizedPath(locale, node.fields.slug),
-            component: slash(
-              node.fields.package ? localPackageTemplate : docsTemplate
-            ),
-            context: {
-              slug: node.fields.slug,
-              locale,
-              ...nextAndPrev,
-            },
-          })
-        }
-      })
-
-      const allPackages = result.data.allNpmPackage.edges
-      // Create package readme
-      allPackages.forEach(edge => {
-        if (_.includes(localPackagesArr, edge.node.title)) {
-          createPage({
-            path: edge.node.slug,
-            component: slash(localPackageTemplate),
-            context: {
-              slug: edge.node.slug,
-              id: edge.node.id,
-              layout: `plugins`,
-            },
-          })
-        } else {
-          createPage({
-            path: edge.node.slug,
-            component: slash(remotePackageTemplate),
-            context: {
-              slug: edge.node.slug,
-              id: edge.node.id,
-              layout: `plugins`,
-            },
-          })
-        }
-      })
-
-      // Create feature comparison pages
-      const jamstackPages = generateComparisonPageSet(`jamstack`)
-      const cmsPages = generateComparisonPageSet(`cms`)
-      const comparisonPages = [...jamstackPages, ...cmsPages]
-      for (const { path, options, featureType } of comparisonPages) {
-        createPage({
-          path,
-          component: slash(featureComparisonPageTemplate),
-          context: {
-            options,
-            featureType,
-          },
-        })
-      }
+  // Create feature comparison pages
+  const jamstackPages = generateComparisonPageSet(`jamstack`)
```

Comment:
features probably goes in its own file anyways, but we can do this in another PR.